.github/SUPPORT.md

```diff
@@ -0,0 +1,27 @@
+## Support for scikit-image
+
+Welcome to scikit-image! We use GitHub for tracking bugs and feature requests.
+GitHub [Issues](https://github.com/scikit-image/scikit-image/issues)
+should be used for reporting bugs or performance regressions. We prefer use of
+GitHub [Discussions](https://github.com/scikit-image/scikit-image/discussions)
+rather than issues for feature requests. If you need support for using
```

Comment:
```suggestion
When you report an [issue](https://github.com/scikit-image/scikit-image/issues),
you may apply a label to it to specify its type: bug, feature request, performance,
infrastructure, documentation, etc.
If you need support for using
```
Right? I guess we can mark this draft PR as 'ready for review' :slightly_smiling_face: